src/components/tools/Location/StaticMap.svelte

```diff
@@ -15,26 +16,13 @@
   export let style = "mapbox/streets-v11";
   export let padding = 50;
   export let zoom = 8;
+  export let useButton = true;
 
   const { accessToken } = mapboxgl;
   const MAX_IMG_HEIGHT = 250;
 
-  const image = new Image();
-  image.onload = () => {
-    loading = false;
-    loaded = true;
-  };
-  image.onerror = () => {
-    loading = false;
-    error = true;
-    logException(
-      `StaticMap image failed for ${location && location.title} at ${
-        location && location.center && location.center.join(",")
-      }`
-    );
-  };
-
-  let loading = true;
+  let imageWrapper;
+  let image;
   let loaded = false;
```

Comment:
these variables (`loaded`, `error`) could be replaced with a single variable, e.g. `state` that is set to a string, e.g. "pending", "loaded", "error" to make the error message display when the static maps API call fails. Currently, when the API returns an error, the map will show the previous location and not the error message.